ws-nextjs-app/package.json

```diff
@@ -22,21 +22,23 @@
     "test:e2e": "yarn stop && yarn build && run-p --race start cypress -- --e2e",
     "test:e2e:interactive": "yarn stop && yarn build && run-p --race start cypress:interactive -- --e2e"
   },
+  "peerDependencies": {
+    "@babel/core": "*",
+    "@babel/plugin-proposal-export-default-from": "*",
+    "@babel/preset-react": "*",
+    "@emotion/babel-plugin": "*",
+    "@emotion/jest": "*",
+    "@testing-library/jest-dom": "*",
+    "@testing-library/react": "*",
+    "cypress": "*",
+    "cypress-terminal-report": "*",
+    "jest-environment-jsdom": "*",
+    "npm-run-all2": "*"
+  },
   "devDependencies": {
-    "@babel/core": "7.25.2",
-    "@babel/plugin-proposal-export-default-from": "7.24.7",
-    "@babel/preset-react": "7.24.7",
-    "@emotion/babel-plugin": "11.11.0",
-    "@emotion/jest": "^11.11.0",
-    "@testing-library/jest-dom": "6.4.5",
-    "@testing-library/react": "15.0.7",
     "babel-plugin-transform-rename-import": "2.3.0",
-    "cypress": "13.10.0",
-    "cypress-terminal-report": "6.0.2",
-    "jest": "^29.7.0",
-    "jest-environment-jsdom": "^29.7.0",
+    "jest": "*",
     "next-test-api-route-handler": "^4.0.8",
```

Comment:
Should we consider not using `^` notation, and sticking to a fixed version (which is the pattern we seem to follow in the main `package.json` file?)